e2e/pos_test.go

```diff
@@ -613,3 +615,171 @@ func TestPoS_StakeUnstakeWithinSameBlock(t *testing.T) {
 
 	validateValidatorSet(senderAddr, client, t, false, numDummyStakers)
 }
+
+func getSnapshot(
+	client ibftOp.IbftOperatorClient,
+	blockNum uint64,
+	ctx context.Context,
+) (*ibftOp.Snapshot, error) {
+	snapshot, snapshotErr := client.GetSnapshot(ctx, &ibftOp.SnapshotReq{
+		Latest: false,
+		Number: blockNum,
+	})
+
+	return snapshot, snapshotErr
+}
+
+func getNearestNextEpochBlock(blockNum uint64, epochSize uint64) uint64 {
```

Comment:
can be renamed to simply `getNextEpochBlock`

Reply:
Fixed 👍 

d2cc7ff